OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs

```diff
@@ -391,105 +391,105 @@ void SelectServer(GameServer server)
 
 		void RefreshServerListInner(List<GameServer> games)
 		{
-			if (games == null)
-				return;
-
-			var mods = games.GroupBy(g => g.Mods)
-				.OrderByDescending(g => GroupSortOrder(g.First()))
-				.ThenByDescending(g => g.Count());
-
 			ScrollItemWidget nextServerRow = null;
 			var rows = new List<Widget>();
-			foreach (var modGames in mods)
-			{
-				if (modGames.All(Filtered))
-					continue;
-
-				var header = ScrollItemWidget.Setup(headerTemplate, () => true, () => { });
-
-				var headerTitle = modGames.First().ModLabel;
-				header.Get<LabelWidget>("LABEL").GetText = () => headerTitle;
-				rows.Add(header);
 
-				Func<GameServer, int> listOrder = g =>
+			if (games != null)
```

Comment:
Our code style is to prefer early return over excessive levels of if statements.  What was your goal here?